### PR TITLE
Closes #16. Add ability to use SSL

### LINKED ISSUE #16: Add ability to use SSL
Add the ability to use SSL to encrypt requests traffic if consul is fronted by an SSL offloading load balancer.

### PR DESCRIPTION
**Pull request by pete0emerson**
Here's my stab at closing #16.

### CODE REVIEW

File: consul/base.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def __init__(
             host='127.0.0.1',
             port=8500,
             token=None,
+            ssl=False,
```
Comment (cablehead) [inline consul/base.py]: How about scheme='http' instead?, Customizable to be 'https' .. not sure why I think that seems better :) Just a vibe.

**Comment by cablehead** [comment]
Is it possible to patch tornado.py as well?

**Comment by pete0emerson** [comment]
Ahh, I didn't touch tornado.py. I'm not a tornado user, but I'll take a look, the change may be similarly easy. And I'll change to your scheme='http', sounds good to me.

**Comment by pete0emerson** [comment]
Do you prefer "scheme" or "transport" ?

**Comment by cablehead** [comment]
I think, marginally, "scheme", as we're using it to swap out the first part
of the URI

http://en.wikipedia.org/wiki/URI_scheme

**Comment by pete0emerson** [comment]
Okay, I moved it to scheme and took a stab at tornado support. I tested default, specifying scheme='http', and specifying scheme='https'.

**Comment by cablehead** [comment]
Thanks Pete!

**Comment by cablehead** [comment]
Pushed to PyPi as version 0.3.6

**Pull request merged**